storage_controller/src/service.rs

```diff
@@ -4969,6 +4969,8 @@ impl Service {
                     cancel: cancel.clone(),
                 });
 
+                let span = tracing::info_span!(parent: None, "drain_node", %node_id);
```

Comment:
`parent: None` disconnects it from the http request handling span, which is sad because we lose the request_id which initiated this. however, in general, we are not doing good at all with caused_by (request_id) and correlation_id (non-existent in our repo) so this is probably fine for now.